Defs/RecipeDefs/LotRE_Recipes.xml

```diff
@@ -4,10 +4,10 @@
 
    <!-- Without vegetable garden, Lembas bread is produced with lembas cornmeal. -->
    <RecipeDef>
-	<defName>LotRE_BakeElvishBread</defName>
```

Comment:
Just a reminder of what I mentioned in the chatroom.
DefNames must not be changed.
Changing a DefName will cause existing games to crash and burn in a firey ball of deathmetal.

Reply:
Ok I think we're good here now. All the defNames should be the originals again.